cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/extensions/CgmesSshControlAreas.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.extensions;
+
+import com.powsybl.cgmes.conversion.extensions.CgmesSshControlAreasImpl.ControlArea;
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Network;
+
+import java.util.List;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public interface CgmesSshControlAreas extends Extension<Network> {
```

Comment:
I am really not sure about this extension as we don't have the use case.Maybe we have to accept to not support these fields in the beginning.